server/src/main/java/org/elasticsearch/plugins/DiscoveryPlugin.java

```diff
@@ -85,4 +86,8 @@
      * {@link IllegalStateException} if the node and the cluster-state are incompatible.
      */
     default BiConsumer<DiscoveryNode,ClusterState> getJoinValidator() { return null; }
+
+    default Map<String, ElectionStrategy> getElectionStrategies() {
```

Comment:
Please add at least a basic javadoc. I would reference here the setting that is used to set the election strategy.

Reply:
fixed in e284426